packages/hardhat-core/src/internal/hardhat-network/provider/utils/convertToEdr.ts

```diff
@@ -258,6 +260,19 @@ export function ethereumjsHeaderDataToEdrBlockOptions(
   };
 }
 
+export function ethereumjsIntervalMiningConfigToEdr(
+  config: IntervalMiningConfig
+): number | IntervalRange {
+  if (isNumber(config)) {
```

Comment:
I try to avoid using lodash if possible, since we'll probably remove it as a dependency eventually. You can just do this:

```suggestion
  if (typeof config === "number") {
```

Reply:
Addressed in https://github.com/NomicFoundation/hardhat/pull/4619